pokemongo_bot/step_walker.py

```diff
@@ -0,0 +1,36 @@
+import logger
+
+from cell_workers.utils import distance, i2f, format_time
+from human_behaviour import random_lat_long_delta, sleep
+from math import ceil
+
+
+class StepWalker(object):
+
+    def __init__(self, bot):
+        self.bot = bot
+        self.api = bot.api
+
+    def step(self, speed, lat, lng):
+        if self.api._position_lat == lat and self.api._position_lng == lng:
+            return True
+
+        dist = distance(
+            i2f(self.api._position_lat),
+            i2f(self.api._position_lng),
+            lat,
+            lng
+        )
+
+        dLat = (lat - i2f(self.api._position_lat))
+        dLng = (lng - i2f(self.api._position_lng))
```

Comment:
Just change this to
(dLat, dLng) = ((lat - i2f(self.api._position_lat)), (lng - i2f(self.api._position_lng)))`